src/runtime/pkg/device/drivers/block.go

```diff
@@ -159,13 +158,13 @@ func (device *BlockDevice) GetDeviceInfo() interface{} {
 }
 
 // Save converts Device to DeviceState
-func (device *BlockDevice) Save() persistapi.DeviceState {
+func (device *BlockDevice) Save() config.DeviceState {
 	ds := device.GenericDevice.Save()
 	ds.Type = string(device.DeviceType())
 
 	drive := device.BlockDrive
 	if drive != nil {
-		ds.BlockDrive = &persistapi.BlockDrive{
+		ds.BlockDrive = &config.BlockDriveState{
```

Comment:
So we have unified the Configuration and State to be one? Did we always have `BlockDriveState`

Reply:
It was called BlockDrive when it was unique to the persist API package.  I needed to rename to avoid conflict when pulling it into device/drivers package. The third commit ends up reconciling this.